files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -303,11 +303,8 @@
 #
 # Customize the GCP linux environment.
 #
-# Update the override file for the GCP instance. This file gets
-# applied dynamically by running google_instance_setup script.
-#
 - blockinfile:
-    path: /etc/default/instance_configs.cfg.template
```

Comment:
Do we need to undo any configuration that might have previously been applied to the `.template` file? E.g. remove the file? or remove the block we may have added in an earlier version?

Reply:
I had a meeting with google to discuss how the new packages changed behavior and they mentioned that they maintained the template file in the new packages too even though it's not documented. I decided that it would be best to remove both the template file and the cfg file and just recreate it as part of our ansible logic.